cmake/sca/cpptest/sca.cmake

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024, Space Cubics, LLC.
+
+find_program(CPPTESTSCAN cpptestscan REQUIRED)
+message(STATUS "Found Parasoft C++test: ${CPPTESTSCAN}")
```

Comment:
looks like the official name of the product is "Parasoft C/C++test" so it would make sense to stick to this across the entire PR (unless Parasot C++test is a thing and I just missed it)

Reply:
Thanks.  Fixed.

Would it be nice if the line reads something like: `Found SCA: Parasoft C/C++test (/usr/bin/cpptestscan)`? 

- It aligns with other "Found" messages
- It tells what the Parasoft C/C++test is by the string `SCA`

If you like it, I can add another commit to chnage the lines from codechecker and sparse.

Reply:
I've added a commit to add "SCA" string for other SCA tools.  If you don't like it, please drop the commit.  Thanks.